app/services/search/google/search_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Search
+  module Google
+    class SearchService < Search::BaseService
+      GOOGLE_SEARCH_BASE_URL = 'https://www.google.com/search'
+
+      def initialize(keyword, language = 'en')
+        super(keyword, language)
+      end
+
+      def search
+        uri = URI.parse("#{GOOGLE_SEARCH_BASE_URL}?q=#{keyword}&hl=#{language}&lr=#{language}")
+
+        response = Net::HTTP.get_response(uri)
```

Comment:
Just curious, any reason we didn't use Nokogiri here? 🙏 

It will be beneficial once we start scraping the actual HTML

Reply:
@Tuubz `Nokogiri` is supposed for parsing HTML content. With the feature scope, we won't parse any the HTML to get the values so we don't need to use it.

Reply:
You are correct that we won't parse the HTML content yet.

But if I'm not mistaken we can make get the response with Nokigiri, to then later parse the HTML later on

```
require 'nokogiri'
require 'open-uri'

# Fetch and parse HTML document
doc = Nokogiri::HTML(URI.open('https://nokogiri.org/tutorials/installing_nokogiri.html'))

# Search for nodes by css
doc.css('nav ul.menu li a', 'article h2').each do |link|
  puts link.content
end
```

Reference: https://github.com/sparklemotion/nokogiri#parsing-and-querying

Reply:
Oh, I see my mistake now.

So in the next issue we can call `Nokogiri::HTML(response)`, right? 🙏

Reply:
> So in the https://github.com/nimblehq/ic-rails-huey-toby/issues/15 we can call Nokogiri::HTML(response.body), right? 🙏

Yup 👍